notebooks/access_github_ci_data.ipynb

```diff
@@ -68,17 +68,30 @@
     "TOKEN = os.getenv(\"GITHUB_ACCESS_TOKEN\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "842fbe80-3d54-4dfb-83c3-c18c5cdcda89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# mode can be \"check or \"workflow\" depending on whether you want to collect checks data or workflows data\n",
+    "MODE = \"check\""
+   ]
+  },
   {
    "cell_type": "markdown",
    "id": "8b7eb146-b77a-4f3e-be81-906a9cf92885",
    "metadata": {},
    "source": [
-    "For example, lets collect data for the test ID 28698040 for the workflow runs in the repository `oss-aspen/8Knot` https://api.github.com/repos/oss-aspen/8Knot/actions/workflows"
+    "* Workflows: For example, lets collect data for the test ID 28698040 for the workflow runs in the repository `oss-aspen/8Knot` https://api.github.com/repos/oss-aspen/8Knot/actions/workflows\n",
+    "\n",
+    "* Checks: Get commits for a repo and get check runs for each commit"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 4,
```

Comment:
Is there a function you can write like `get_test_ids(repo_name)` ? Or does it make sense to assume the user will know these for the tests they are interested in?

Reply:
Is there any reason to not make this function more general? `get_page_numbers(test_id, org, repo)` instead of hard-coding the repo of interest here?